<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Port_Window</class>
 <widget class="QDialog" name="Add_Port_Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>464</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adding Port</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/port.png</normaloff>:/port.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="Label_Port">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Redirect virtual port to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="RB_Default">
         <property name="text">
          <string>Default</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_vc">
         <property name="text">
          <string>vc (Virtual console)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_pty">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>pty (Pseudo TTY)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_none">
         <property name="text">
          <string>none</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_null">
         <property name="text">
          <string>null (/dev/null)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_Dev">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>/dev/</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="RB_File">
         <property name="text">
          <string>File</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_stdio">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>stdio</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_Pipe">
         <property name="text">
          <string>Pipe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_COM">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>COM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_UDP">
         <property name="text">
          <string>UDP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_msmouse">
         <property name="text">
          <string>msmouse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="RB_TCP">
         <property name="text">
          <string>TCP</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_Telnet">
         <property name="text">
          <string>Telnet</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_Unix">
         <property name="text">
          <string>Unix</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_mon">
         <property name="text">
          <string>mon</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="RB_braille">
         <property name="text">
          <string>braille</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="Label_Args">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Additional Arguments:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QLineEdit" name="Edit_Args"/>
     </item>
     <item>
      <widget class="QToolButton" name="TB_Browse">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/open-file.png</normaloff>:/open-file.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="Label_ArgsHelpS">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Arguments Help:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="Label_Help">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="Button_OK">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/ok.png</normaloff>:/ok.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Button_Cancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="images.qrc">
         <normaloff>:/cancel.png</normaloff>:/cancel.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>RB_Default</tabstop>
  <tabstop>RB_vc</tabstop>
  <tabstop>RB_pty</tabstop>
  <tabstop>RB_none</tabstop>
  <tabstop>RB_null</tabstop>
  <tabstop>RB_Dev</tabstop>
  <tabstop>RB_File</tabstop>
  <tabstop>RB_stdio</tabstop>
  <tabstop>RB_Pipe</tabstop>
  <tabstop>RB_COM</tabstop>
  <tabstop>RB_UDP</tabstop>
  <tabstop>RB_msmouse</tabstop>
  <tabstop>RB_TCP</tabstop>
  <tabstop>RB_Telnet</tabstop>
  <tabstop>RB_Unix</tabstop>
  <tabstop>RB_mon</tabstop>
  <tabstop>RB_braille</tabstop>
  <tabstop>Edit_Args</tabstop>
  <tabstop>TB_Browse</tabstop>
  <tabstop>Button_OK</tabstop>
  <tabstop>Button_Cancel</tabstop>
 </tabstops>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Button_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>Add_Port_Window</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>440</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Button_OK</sender>
   <signal>clicked()</signal>
   <receiver>Add_Port_Window</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>345</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
